src/test/unit/lambda_tracer/test_lambda_reporter.py

```diff
@@ -73,7 +228,67 @@ def test_create_request_body_take_error_first(dummy_span, error_span, function_e
         function_end_span,
     ]
     size = _get_event_base64_size(expected_result)
-    assert _create_request_body(input, True, size) == json.dumps(expected_result)
+    assert _create_request_body(
+        input_spans, True, max_size=size, max_error_size=size
+    ) == json.dumps(expected_result)
+
+
+def test_create_request_body_take_only_metadata_function_span(
+    function_end_span: dict, function_end_span_metadata: dict
+):
+    expected_result = [function_end_span_metadata]
+    input_spans = [function_end_span]
+    size = _get_event_base64_size(expected_result)
+
+    result = _create_request_body(input_spans, True, max_size=size, max_error_size=size)
+
+    assert result == json.dumps(expected_result)
```

Comment:
can you check the log?
If there's no log - then add one! this will make our life much easier when debugging